snap/snapcraft.yaml

```
@@ -263,6 +266,7 @@ parts:
       - libsz2
       - libhdf5-hl-100
       - libhdf5-103-1
+      - util-linux
```

[nit]: `util-linux` brings in a boatload of packages as it's kind of a catch-all for various system utilities used by Linux systems. Figured this additional comment here just states that we're pulling in this package so we can have the `setpriv` utility available inside of the Slurm snap. We do the same thing for `procps` brought in by the `logrotate` part.

```suggestion
      - util-linux  # `setpriv`
```